machine/v1/types_powervsprovider.go

```diff
@@ -121,6 +130,11 @@ type PowerVSMachineProviderConfig struct {
 	// default, which is subject to change over time. The current default is 32.
 	// +optional
 	MemoryGiB int32 `json:"memoryGiB,omitempty"`
+
+	// LoadBalancers is the set of load balancers to which the new instance
+	// should be added once it is created.
+	// +optional
+	LoadBalancers []LoadBalancerReference `json:"loadBalancers,omitempty"`
```

Comment:
What will happen if this is set on a worker machine? Will it still work?
Is there any difference between internat and external LBs in IBM?

Reply:
We have planned to set it to only on control plane machines and even in MAPI we can have check to add loadbalancer for only Master VMs.
In IBM we have public loadbalancer for serving api server traffic which will be served from 6443 port and private for internal communications , it serves on port 6443 and 22623 for machine-config traffic.